### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/1197.

### CODE REVIEW

**Review by kopiczko — APPROVED** [review]
LGTM, but I know very little about circle

**Comment by xh3b4sd** [comment]
Well, it is just copy/paste from some other project so I expect it to be right. :D

File: meta.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package flanneltpr
+
+// Metadata to create operatorkit TPR.
+
+const (
+	// Name represents the name of the third party resource within Kubernetes.
+	Name = "flannel.network.giantswarm.io"
```
Comment (kopiczko) [inline meta.go:7]: @xh3b4sd sorry I didn't catch that earlier. Should be `flannel-network.giantswarm.io`.

Comment (kopiczko) [reply meta.go:7]: Or maybe just `flannel.giantswarm.io`? `kubectl delete flannel` makes sense.

Comment (xh3b4sd) [reply meta.go:7]: I am going with `flannel-network.giantswarm.io`. See https://github.com/giantswarm/flanneltpr/pull/2.

**Pull request merged**